EMULATOR/LIBGPU.C

```diff
@@ -417,6 +458,24 @@ void SetDrawMode(DR_MODE* p, int dfe, int dtd, int tpage, RECT16* tw)
 	setDrawMode(p, dfe, dtd, tpage, tw);
 }
 
+void SetDrawMove(DR_MOVE* p, RECT16* rect, int x, int y)
+{
+	char uVar1;
```

Comment:
Interesting, any chance this could be cleaned before I verify that the code is correct?